__tests__/app.test.js

```diff
@@ -86,3 +87,33 @@ describe("GET", () => {
       });
   });
 });
+
+describe("GET", () => {
+  it("should return array of comments when given review_id with following properties", () => {
+    return request(app)
+      .get(`/api/reviews/2/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments.length).toBe(3);
```

Comment:
this test will fail if any more comments are added when we check the length here